app/lib/common/widgets/redact_content.dart

```diff
@@ -94,44 +94,25 @@ class RedactContentWidget extends ConsumerWidget {
     try {
       if (isSpace) {
         final space = await ref.read(spaceProvider(roomId).future);
-        res = await space.redactContent(eventId, reason);
+        final redactedId = await space.redactContent(eventId, reason);
         debugPrint(
-          'Content from user:{$senderId redacted $res reason:$reason}',
+          'Content from user:{$senderId redacted $redactedId reason:$reason}',
         );
       } else {
         final room = await ref.read(chatProvider(roomId).future);
-        res = await room.redactContent(eventId, reason);
+        final redactedId = await room.redactContent(eventId, reason);
         ref.invalidate(spaceEventsProvider);
         debugPrint(
-          'Content from user:{$senderId redacted $res reason:$reason}',
+          'Content from user:{$senderId redacted $redactedId reason:$reason}',
         );
       }
 
-      if (res) {
-        if (ctx.mounted) {
-          Navigator.of(ctx, rootNavigator: true).pop();
-          Navigator.of(ctx, rootNavigator: true).pop(true);
-          customMsgSnackbar(ctx, 'Content successfully removed');
-          if (onSuccess != null) {
-            onSuccess!();
-          }
-        }
-      } else {
-        if (ctx.mounted) {
-          Navigator.of(ctx, rootNavigator: true).pop();
-          showAdaptiveDialog(
-            context: ctx,
-            builder: (ctx) => DefaultDialog(
-              title: const Text('Removing content failed'),
-              actions: <Widget>[
-                DefaultButton(
-                  onPressed: () => Navigator.of(ctx, rootNavigator: true).pop(),
-                  title: 'Close',
-                  isOutlined: true,
-                ),
-              ],
-            ),
-          );
+      if (ctx.mounted) {
+        Navigator.of(ctx, rootNavigator: true).pop();
+        Navigator.of(ctx, rootNavigator: true).pop(true);
+        customMsgSnackbar(ctx, 'Content successfully removed');
```

Comment:
I'd propose to change all this structure using ```EasyLoading``` indicators, its quite handy in code reduction and easy understanding of what's happening ;)